server/services/user.test.js

```diff
@@ -69,6 +69,7 @@ describe('User service', () => {
       accessRoles,
       staffRoles: [],
       isWhereabouts: false,
+      isUseOfForce: false,
```

Comment:
you'll need some additional tests here:
* One that checks when a prison has been enable
* One that checks when PRISONS variable is empty string

Reply:
done